Sources/InternxtSwiftCore/Services/HttpAPI/NetworkAPI.swift

```diff
@@ -9,20 +9,44 @@ import Foundation
 
 
 @available(macOS 10.15, *)
-struct NetworkAPI {
-    private let apiClient = APIClient()
-    private let configLoader = ConfigLoader()
+public struct NetworkAPI {
+    private let baseUrl: String
+    private let apiClient: APIClient
+
+    public init(baseUrl: String, authToken: String) {
```

Comment:
Why not set the debug flag to the struct instead of to each endpoint? It will be less repetitive. It currently provides more flexibility, but it is the granularity worth the point?

Reply:
usually you want to debug certain endpoints, not the whole struct, that's why is added at the method level, if I add it to the struct and I get a bunch of FileProvider requests, the console will get bloated of responses